ibm/mas_devops/roles/ocp_upgrade/tasks/main.yml

```diff
@@ -0,0 +1,29 @@
+---
+# 1. (optionally) Upgrade OCP to v4.8
+# -----------------------------------------------------------------------------
+# MAS 8.7 is fully supported on both OCP 4.6, 4.8 (we can't support 4.7 because
+# CP4D does not), however at some point before you eventually upgrade to MAS
+# 8.8 you will need to upgrade OCP to v4.8
+
+# We can only really automate the upgrade in ROKS, in other cluster types
```

Comment:
Can we add a condition to the below to only upgrade if the cluster type is roks?

Reply:
We have an assertion for it here:
https://github.com/ibm-mas/ansible-devops/blob/upgrade87/ibm/mas_devops/roles/ocp_upgrade/tasks/ocp_upgrade.yml#L11